website/docs/r/cluster.html.markdown

```diff
@@ -348,6 +348,13 @@ In addition to all arguments above, the following attributes are exported:
 * `mongo_uri` - Base connection string for the cluster. Atlas only displays this field after the cluster is operational, not while it builds the cluster.
 * `mongo_uri_updated` - Lists when the connection string was last updated. The connection string changes, for example, if you change a replica set to a sharded cluster.
 * `mongo_uri_with_options` - connection string for connecting to the Atlas cluster. Includes the replicaSet, ssl, and authSource query parameters in the connection string with values appropriate for the cluster.
+* `connection_strings` - Set of connection strings that your applications use to connect to this cluster.
```

Comment:
Please modify a bit more to the following - these can be tricky for folks so want to pretty much copy the source (minus a few things): 
* `connection_strings` - Set of connection strings that your applications use to connect to this cluster. More info in [Connection-strings](https://docs.mongodb.com/manual/reference/connection-string/).  Use the parameters in this object to connect your applications to this cluster. To learn more about the formats of connection strings, see [Connection String Options](https://docs.atlas.mongodb.com/reference/faq/connection-changes/). NOTE: Atlas returns the contents of this object after the cluster is operational, not while it builds the cluster.

Reply:
Thank you, melissa I'll change it and the rest you suggested